curriculum/challenges/english/25-front-end-development/workshop-fruit-search-app/68140ab94fd0ea19c62ca87e.md

```diff
@@ -0,0 +1,273 @@
+---
+id: 68140ab94fd0ea19c62ca87e
+title: Step 21
+challengeType: 0
+dashedName: step-21
+---
+
+# --description--
+
+Below your `timeoutId` variable, return a cleanup function that clears the timeout using `clearTimeout(timeoutId)`. This prevents multiple delayed fetches from stacking up. Make sure your cleanup function is an arrow function.
```

Comment:
My code didn't pass that I think should:

```
    return () => {
      clearTimeout(timeoutId);
    }
```

We could mention to use an implicit return - or adjust the test to allow various syntax.